### DISCUSSION TITLE
500: Internal Error

### DISCUSSION THREAD

**Discussion by zouzeTG** [open-webui/open-webui#7558] [Issues]
# Bug Report

I can access to Web UI, but when I want to access at others page : (models, knowledge...setting) i have this error : "500: Internal Error"

## Important Notes

- **Before submitting a bug report**: Please check the Issues or Discussions section to see if a similar issue or feature request has already been posted. It's likely we're already tracking it! If you’re unsure, start a discussion post first. This will help us efficiently focus on improving the project.

- **Collaborate respectfully**: We value a constructive attitude, so please be mindful of your communication. If negativity is part of your approach, our capacity to engage may be limited. We’re here to help if you’re open to learning and communicating positively. Remember, Open WebUI is a volunteer-driven project managed by a single maintainer and supported by contributors who also have full-time jobs. We appreciate your time and ask that you respect ours.

- **Contributing**: If you encounter an issue, we highly encourage you to submit a pull request or fork the project. We actively work to prevent contributor burnout to maintain the quality and continuity of Open WebUI.

- **Bug reproducibility**: If a bug cannot be reproduced with a `:main` or `:dev` Docker setup, or a pip install with Python 3.11, it may require additional help from the community. In such cases, we will move it to the "issues" Discussions section due to our limited resources. We encourage the community to assist with these issues. Remember, it’s not that the issue doesn’t exist; we need your help!

Note: Please remove the notes above when submitting your post. Thank you for your understanding and support!

---

## Installation Method

Installation method : git clone https://github.com/open-webui/open-webui.git / npm i / npm run build .....

## Environment

- **Open WebUI Version:** :  v0.4.7 (latest)
- **Ollama (if applicable):**: 0.4.7  (latest)

- **Operating System:** : Windows 2022 server
- **Browser (if applicable):** Edge 

**Confirmation:**

- [ ] I have read and followed all the instructions provided in the README.md.
- [ ] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.
- [ ] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

[Describe what you expected to happen.]

## Actual Behavior:

[Describe what actually happened.]

## Description

**Bug Summary:**
[Provide a brief but clear summary of the bug]

## Reproduction Details

**Steps to Reproduce:**
[Outline the steps to reproduce the bug. Be as detailed as possible.]

## Logs and Screenshots

**Browser Console Logs:**
[Include relevant browser console logs, if applicable]

**Docker Container Logs:**
[Include relevant Docker container logs, if applicable]

**Screenshots/Screen Recordings (if applicable):**
[Attach any relevant screenshots to help illustrate the issue]

## Additional Information

[Include any additional details that may help in understanding and reproducing the issue. This could include specific configurations, error messages, or anything else relevant to the bug.]

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

![Error-500](https://github.com/user-attachments/assets/7183d118-c1dd-4252-b4ab-0fa8c50568f6)

**Comment by zouzeTG** [comment]
![Error-500](https://github.com/user-attachments/assets/b130c261-4367-45b0-b4ae-a738d659f180)
![Error-500-2](https://github.com/user-attachments/assets/3a4117f6-e837-4831-99af-22377034886f)

**Comment by tianjianchn** [comment]
Are you using older version of Firefox? like 89? Check the devtools console to see if there are any errors
